packages/@glimmer/runtime/index.ts

```diff
@@ -114,7 +114,7 @@ export {
 } from './lib/modifier/interfaces';
 
 export { default as DOMChanges, DOMChanges as IDOMChanges, DOMTreeConstruction, isWhitespace, insertHTMLBefore } from './lib/dom/helper';
-import  * as Simple from './lib/dom/interfaces';
+import { Simple } from '@glimmer/interfaces';
```

Comment:
It seems I cannot import and export `@glimmer/interfaces` from this file. 

Error:
 
```
Could not find module `@glimmer/interfaces` imported from `@glimmer/runtime/index`
```